hummingbot/data_feed/candles_feed/candles_base.py

```diff
@@ -68,6 +68,7 @@ async def start_network(self):
         """
         This method starts the network and starts a task for listen_for_subscriptions.
         """
+        self.logger().info("Starting candles feed...")
```

Comment:
I think that is worth to add the exchange and trading pair that you are starting the candles from

Reply:
I may have put that as a debug. I have updated it